python/py-sip/Portfile

```diff
@@ -67,15 +67,28 @@ if {${name} ne ${subport}} {
 
     depends_lib-append \
                     port:py${python.version}-packaging \
-                    port:py${python.version}-setuptools \
-                    port:py${python.version}-toml \
-                    port:py${python.version}-tomli
+                    port:py${python.version}-setuptools
 
     if {${python.version} < 38} {
         depends_lib-append \
                     port:py${python.version}-importlib-metadata
     }
 
+    if {${python.version} <= 36} {
+        depends_lib-append \
+                    port:py${python.version}-toml
+    } elseif {${python.version} < 311} {
+        depends_lib-append \
+                    port:py${python.version}-tomli
+    }
+
+    if {[string match "*-sip-devel" ${subport}]} {
+        depends_lib-append \
+                    port:py${python.version}-toml
+        depends_lib-delete \
+                    port:py${python.version}-tomli
+    }
```

Comment:
sorry for the delay, I'll check Nx merge tonight. I an a bit confused though why the "-devel" port would only use the "toml" port though, but will verify this evening

Reply:
> sorry for the delay, I'll check Nx merge tonight. I an a bit confused though why the "-devel" port would only use the "toml" port though, but will verify this evening

No worries, this is not urgent. The devel port is pinned on version 6.6.2, which requires toml (https://www.riverbankcomputing.com/hg/sip/file/a3c9672f761a/setup.py).

Reply:
Another question is whether the devel port (in particular pinned at an old version) is needed at all anymore?

Reply:
aha, that's where my confusion came from - the `-devel` port is supposed to follow upstream development (i.e., most often a tag instead of a released version) so pining that to an earlier version is kind of pointless.
We can leave it for now but should do something about that in a future update/change ;)

Thanks for the PR - I'll merge this!